src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Log/FunctionId.cs

```diff
@@ -594,5 +594,21 @@ internal enum FunctionId
 
         // 680-690 LSP Initialization info ids.
         LSP_Initialize = 680,
+
+        // 690-729 Performance histogram logging.
```

Comment:
I'd even go more disparate and start from 700, and reserve the entire range (say 700-800) for histogram function IDs. I like the fact that you have used distinct suffixes `Delay` and `Summary`, a comment to indicate what each of these two buckets indicates might also be helpful here. Also do we expect all these IDs to be in Delay/Summary pairs?

Reply:
Updated the comment with a bit more information. The intent was to use Delay/Summary pairs (even if one of them isn't used). Also updated the range per your suggestion. Commit 3.